File: C/bubblesort/meta-llama-llama-4-maverick-17b-128e-instruct.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

void bubbleSort(int arr[], int n) {
    int temp;
    for (int i = 0; i < n - 1; i++) {
        for (int j = 0; j < n - i - 1; j++) {
            if (arr[j] > arr[j + 1]) {
                // Swap arr[j] and arr[j+1]
                temp = arr[j];
                arr[j] = arr[j + 1];
                arr[j + 1] = temp;
            }
        }
    }
}

void generateRandomArray(int arr[], int n) {
    for (int i = 0; i < n; i++) {
        arr[i] = rand() % 1000;
    }
}

int main() {
    srand(time(NULL)); // seed for random number generation
    int n = 1000;
    int* arr = (int*)malloc(n * sizeof(int));
    generateRandomArray(arr, n);
    bubbleSort(arr, n);
    free(arr);
    return 0;
}

File: Java/bubblesort/meta_llama_llama_4_maverick_17b_128e_instruct.java
package Java.bubblesort;
import java.util.Random;
public class meta_llama_llama_4_maverick_17b_128e_instruct {
        public static void main(String[] args) {
        int[] arr = new int[1000];
        Random rand = new Random();
        for (int i = 0; i < 1000; i++) {
            arr[i] = rand.nextInt(1000);
        }
        bubbleSort(arr);
    }

    public static void bubbleSort(int[] arr) {
        int n = arr.length;
        for (int i = 0; i < n - 1; i++) {
            boolean swapped = false;
            for (int j = 0; j < n - i - 1; j++) {
                if (arr[j] > arr[j + 1]) {
                    // Swap arr[j] and arr[j+1]
                    int temp = arr[j];
                    arr[j] = arr[j + 1];
                    arr[j + 1] = temp;
                    swapped = true;
                }
            }
            if (!swapped) {
                break;
            }
        }
    }
}
